tests/test_args.py

```diff
@@ -0,0 +1,64 @@
+"""
+Test argument parsing and r2d construction
+"""
+import os
+import pytest
+import logging
+from repo2docker.__main__ import make_r2d
+from repo2docker import __version__
+from contextlib import redirect_stdout
+import io
+
+def test_version():
+    """
+    Test passing '--version' to repo2docker
+    """
+    stdout = io.StringIO()
+    with redirect_stdout(stdout):
```

Comment:
how does `redirect_stdout` compare to https://docs.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function? I've used the latter before I think.

Reply:
@betatim oo, that's pretty nice. I've switched to using that.